src/components/molecules/ChatList/ChatList.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState } from "react";
+import { User } from "types/User";
+import UserProfileModal from "components/organisms/UserProfileModal";
+import {
+  PrivateChatMessage,
+  RestrictedChatMessage,
+} from "components/context/ChatContext";
+import { useSelector } from "hooks/useSelector";
+import { WithId } from "utils/id";
+import { Modal } from "react-bootstrap";
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+import { formatUtcSeconds } from "utils/time";
+import { getLinkFromText } from "utils/getLinkFromText";
+import { useUser } from "hooks/useUser";
+import "./ChatList.scss";
```

Comment:
Noted, will honour that for future reviews as well.

Reply:
I like the idea but this has to be an eslint option. It will be a pain in the ass to order these when you don't remember the order or just simply add a new component with the auto import from your IDE

Reply:
I'll just add them manually for now.

Reply:
> I like the idea but this has to be an eslint option.

100%, this is more of a 'for future keeping things clean', and automatically handled and/or enforced is optimal.

> I'll just order them manually for now.

Appreciated :) But in general for review comments I mark as NB (non-blocking) and nit, they aren't things I am 'requiring' for the PR (aka: I won't block a PR based on them), more just documenting my thought process as I go through the PR/etc.

Reply:
resolved in 77fc5d6d1829a42da2152d00871cde0b6f47269d